Cargo.toml

```diff
@@ -4,13 +4,18 @@ description = "a boltz exchange client for swaps between BTC/LBTC & LN"
 authors = ["i5hi <ishi@satoshiportal.com>", "Rajarshi Maitra <raj@bitshala.org>"]
 version = "0.2.0"
```

Comment:
Might be worth bumping the version now the WS message structs have changed

Reply:
Done [662ab3b](https://github.com/SatoshiPortal/boltz-rust/pull/91/commits/662ab3b85149560b4405b437a5cfe74ff49c39d6)